mysql/README.md.jinja

```diff
@@ -34,16 +34,19 @@ returned by a `SHOW STATUS` command. This includes the following:
 
 ### Creating a MySQL user for collectd
 
-If you're monitoring a secured MySQL deployment, it is a good practice to create a MySQL user with minimal read-only roles, as follows:
+To create a MySQL user for collectd:
 
 ```
-db.createUser( {
-  user: "collectd",
-  pwd: "collectd",
-  roles: [ { role: "readAnyDatabase", db: "admin" }, { role: "clusterMonitor", db: "admin" } ]
-});
+ CREATE USER 'collectd'@'localhost' IDENTIFIED BY 'Fat4ohse';
```

Comment:
We probably want to use `IDENTIFIED BY 'collectdUserPassword';` instead of suggesting something in docs.